DAOS-12221 test: Fix dmg stop/start.

jgmoore-or opened the pull request:
### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

jgmoore-or committed:
DAOS-12221 test: Update method for stopping and starting ranks in tests.

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Change daos_test to use json api for dmg stop/start.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Change invocation of dmg stop in rebuild tests.

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Change invocation of dmg stop for rebuild tests.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Change implementation of dmg stop and dmg start.

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Change implementation of dmg stop.

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOs-12221 test: Change implementation of dmg stop.

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Change implementation of dmg stop.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Change invocation of dmg stop.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-1221 test: Change implementation of dmg stop.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
Merge branch 'master' into jgm/daos-12221-2

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x

jgmoore-or committed:
Merge branch 'master' into jgm/daos-12221-2

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x

jgmoore-or committed:
DAOS-12221 test: Update dmg invocation of stop/start of ranks.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Fix invocation of dmg stop/start.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-leap15: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider-hw-medium: ucx+dc_x
Test-provider-hw-large: ucx+dc_x

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Update implementation of dmg stop/start in daos_test.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-leap15: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-verbs-provider: true
Test-tag: pr daily_regression
Test-provider-hw-large: ucx+dc_x
Test-tag: pr,hw,large daily_regression,hw,large

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Fix dmg stop error.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-verbs-provider: true
Test-tag: pr daily_regression
Test-provider-hw-large: ucx+dc_x
Test-provider-hw-mediaum: ucx+dc_x
Test-tag: pr,hw,large daily_regression,hw,large

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
Merge branch 'master' into jgm/daos-12221-2
Required-githooks: true

jgmoore-or committed:
Merge branch 'master' into jgm/daos-12221-2

jgmoore-or committed:
DAOS-12221 test: Fix dmg system stop/start for rebuild tests.

Merge branch 'master' into jgm/daos-12221-2

jgmoore-or committed:
DAOS-12221 test: Fix dmg stop/start.

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Fix dmg stop/start in test code.

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12221 test: Update dmg stop/start.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11120/1/execution/node/745/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11120/1/execution/node/578/log

jgmoore-or committed:
DAOS-12221 test: Fix dmg stop.

Test-provider: ucx+tcp

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

jgmoore-or committed:
DAOS-12221 test: Fix dmg stop.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+tcp

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

jgmoore-or committed:
DAOS-12221 test: Fix dmg stop.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+tcp

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-11120/4/display/redirect

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11120/4/execution/node/714/log
